### ISSUE TITLE
[Screen Reader – BotKit – Docs]: Table Summary is not defined for all the tables in the pages.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#11]
**User Experience:**
Users who rely on screen readers will affect to navigate, if Table summary and caption is not defined for all the tables in the page.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Docs” link and select it.
Step 3: Navigate to “Core Features” link and select it.
Step 4: Navigate to all the tables which are present in the page.
Step 5: Verify whether Table Summary and Captions are defined for the tables in the page.

**Actual Result:**
Table Summary and Captions are not defined for all the tables in the pages.

**Expected Result:**
Table Summary and Captions should be defined for all the tables in the pages.

**Note**: Same issue repros for all the tables which are present in pages.

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}
![MAS4 2 3_BotKit_Docs_Table Summary](https://user-images.githubusercontent.com/48272005/54471697-1eb77a80-47e3-11e9-9b11-2af39c441a12.png)

**Comment by benbrown** [comment]
@awalia13 due to the fact that these documents are generated from Github markdown, there is not currently a way to add caption tags to the tables.

I would like to request an exception on this one.

**Comment by benbrown** [comment]
Request type: | Exception
-- | --
Leadership approval: | <List GPM or Dev Mgr. who has approved this request> NOTE: Attach approval communication to the request.  GPM or Dev Mgr should have conveyed in the communication that s/he understands the adverse impact to the experience.
Submitter (Accessibility Driver): | ​benbro@microsoft.com
UI / App name: | Botkit Website
User impact: | Users who are using a screen reader may have trouble navigating the content of the tabular data that is contained within the documentation for Botkit.
Repro steps: | See Above
Provide work-around, if any: | No
Mitigating circumstances, if any: | These tables are in-line and are near to other headers and descriptive material that should make their meaning and purpose clear even with the lack of specific table captions.
Regression from past release: | no
Justification: | These tables are automatically generated with Github's markdown rendering system. Markdown does not support table captions, thus it is currently impossible to generate HTML that includes them.
ETA to fix: | No ETA.
